evalml/tests/integration_tests/test_time_series_integration.py

```diff
@@ -61,3 +61,70 @@ def test_can_run_automl_for_time_series_with_categorical_and_boolean_features(
     )
     # Treat all features as not known-in-advanced
     automl.best_pipeline.predict(X_valid, X_train=X, y_train=y)
+
+
+@pytest.mark.parametrize(
+    "problem_type",
+    [
+        ProblemTypes.TIME_SERIES_BINARY,
+        ProblemTypes.TIME_SERIES_MULTICLASS,
+        ProblemTypes.TIME_SERIES_REGRESSION,
+    ],
+)
+def test_can_run_automl_for_time_series_known_in_advance(
+    problem_type,
+):
+
+    X = pd.DataFrame(
+        {"features": range(101, 601), "date": pd.date_range("2010-10-01", periods=500)}
+    )
+    y = pd.Series(range(500))
```

Comment:
super-nit: perhaps consolidate the periods into a var and refactor the range calls  and even the generation of the target to use that var?